usecases/cluster/delegate.go

```diff
@@ -268,7 +271,8 @@ func (d *delegate) updater(period, minPeriod time.Duration, du func(path string)
 		}
 		space, err := du(d.dataPath)
 		if err != nil {
-			d.log.WithField("action", "delegate.local_state.disk_usage").Error(err)
+			d.log.WithField("action", "delegate.local_state.disk_usage").WithError(err).
+				Error("du func error in updater")
```

Comment:
**typo (non-blocking):**  i thought at the beginning it's a typo :D , but it turns it's the name of the argument, maybe we do in the log add the original name which i think `disk space updater`